packages/atomic/src/components/atomic-sort-criteria/atomic-sort-criteria.tsx

```diff
@@ -27,5 +30,27 @@ export class AtomicSortCriteria {
    *
    * You can specify multiple sort criteria to be used in the same request by separating them with a comma (e.g., `criteria="size ascending, date ascending"` ).
    */
-  @Prop() criteria!: string;
+  @Prop() public criteria!: string;
+
+  constructor() {
+    try {
+      new Schema({
+        caption: new StringValue({emptyAllowed: false, required: true}),
```

Comment:
Stencil documentation has this topic about prop validation: 
https://stenciljs.com/docs/properties#prop-default-values-and-validation

They seem to document that we should be doing this with a `@Watch` decorator, and not in the constructor.

Also, I understand the need for caption validation since it is purely a UI concern, but why the criteria validation ?

I think it's already done by the headless controller, and we otherwise risk to have some incompatibility between the two 🤔